src/stpipe/log.py

```diff
@@ -280,18 +280,24 @@ def log_records(self):
         return self._log_records
 
     def emit(self, record):
-        self._log_records.append(record)
+        if not self.formatter:
+            return
+        self._log_records.append(self.formatter.format(record))
```

Comment:
can we make this more explicit?
```suggestion
        if self.formatter is not None:
            self._log_records.append(self.formatter.format(record))
```

Reply:
Thanks!